src/server/config/__tests__/fixtures/run_kbn_server_startup.js

```diff
@@ -0,0 +1,10 @@
+import { createServer } from '../../../../test_utils/kbn_server';
+
+const server = createServer(JSON.parse(process.env.CREATE_SERVER_OPTS));
+
+// We just need the server to run through startup so that it will
+// log the deprecation messages. Once it has started up we close it
+// to allow the process to exit naturally
+server.ready().then(() => {
```

Comment:
__optional nit:__ Wondering if we should also call `close` in case `ready()` promise is rejected? Not a big deal, but maybe handy when someone tries to reuse this fixture and passes wrong arguments to the `kbnServer` so that `ready` is rejected, in this case test may never receive `exit` event.

Reply:
good catch @azasypkin